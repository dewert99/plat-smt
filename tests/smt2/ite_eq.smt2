(declare-sort I)
(declare-const r I)
(declare-const i Bool)
(declare-const t I)
(declare-const e I)
(push)
(assert (= r (ite i t e)))
(check-sat)
(get-model)
(pop)
(assert (not (= r (ite i t e))))
(check-sat)
(get-model)
