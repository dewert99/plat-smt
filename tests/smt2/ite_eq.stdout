sat
(
 (define-fun e () I (as @v3 I))
 (define-fun i () Bool false)
 (define-fun r () I (as @v3 I))
 (define-fun t () I (as @v4 I))
)
sat
(
 (define-fun e () I (as @v7 I))
 (define-fun i () Bool false)
 (define-fun r () I (as @v3 I))
 (define-fun t () I (as @v4 I))
)
